### PR TITLE
Fixed-pass-an-array-as-props

### LINKED ISSUE #39263: React: Pass an Array as Props Bug
**Describe your problem and how to reproduce it:**
In the Front End Libraries Certification, there is a sub-unit in React portion named "React: Pass an Array as Props", in which we have to pass an array as props and to render them in p tag by using  .join() method. The question asks us to display the array in the p element as a comma-separated list. The problem is that if we use .join(",") then the test fails but if we use .join(", ") then the test is passed. I think the first (using .join(",")) should also pass the test. 
I will love to work on this issue if someone tells me the file which has the logic which tests the test. I think it needs a small tweaking which I can do !

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/front-end-libraries/react/pass-an-array-as-props

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 
Google Chrome Dev 
85.0.4183.16 (Official Build) dev (64-bit) (cohort: Dev-nonPGO)
Windows

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
fcc-bug

### PR DESCRIPTION
**Pull request by Himanshunitrr**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39263

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/react/pass-an-array-as-props.english.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ tests:
...
   - text: The <code>List</code> component should render the value from the <code>tasks</code> prop in the <code>p</code> tag.
-    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(ToDo)); return mockedComponent.find('p').get(0).props.children === mockedComponent.find('List').get(0).props.tasks.join(', ') && mockedComponent.find('p').get(1).props.children === mockedComponent.find('List').get(1).props.tasks.join(', '); })());
+    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(ToDo)); return ( mockedComponent.find('p').get(0).props.children === mockedComponent.find('List').get(0).props.tasks.join(', ') || mockedComponent.find('p').get(0).props.children === mockedComponent.find('List').get(0).props.tasks.join(',')) && (mockedComponent.find('p').get(1).props.children === mockedComponent.find('List').get(1).props.tasks.join(', ') || mockedComponent.find('p').get(1).props.children === mockedComponent.find('List').get(1).props.tasks.join(',')); })());
```
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/react/pass-an-array-as-props.english.md:50]: To simplify the code (not so many ||s) and to allow for any number of spaces before or after the comma, I suggest the following:
```suggestion
    testString: |
      assert((function() {
        const mockedComponent = Enzyme.mount(React.createElement(ToDo));
        return (
          mockedComponent.find('p').get(0).props.children.replace(/\s*,\s*/g,',') === mockedComponent.find('List').get(0).props.tasks.join(',').replace(/\s*,\s*/g,',') &&
          mockedComponent.find('p').get(1).props.children.replace(/\s*,\s*/g,',') === mockedComponent.find('List').get(1).props.tasks.join(',').replace(/\s*,\s*/g,',')
        );
      })());
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@Himanshunitrr See my suggested changes.

**Comment by Himanshunitrr** [comment]
I was too thinking to use regex, Thanks for your valuable review.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Now it looks good to me.

**Review by Manish-Giri — APPROVED** [review]
LGTM 💯 

@Himanshunitrr For future PRs, please include the corresponding issue # (if there is one) in the PR body, so that it gets closed when the PR is merged. I have edited this one to include it. 

Thanks for your contribution!

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**